examples/with-styletron/README.md

```diff
@@ -0,0 +1,36 @@
+
+# Styletron example
+
+## How to use
+
+Download the example:
```

Comment:
We changed the structure of the readmes https://github.com/zeit/next.js/pull/485/files#diff-481c8b6dd98d2038ce05b93afea6b60d